### PR TITLE
Merge pull request #5782 from smartcontractkit/chore/vrf-listener-v1-…

### PR DESCRIPTION
**Pull request by Nic0S**
Merge pull request #5782 from smartcontractkit/chore/vrf-listener-v1-fulfill-log
    
VRF V1: Add a log message on successful pipeline run
    
(cherry picked from commit aeb98d85f5f0fc43c9a66debd281c663cfa1149d)

### CODE REVIEW

File: core/services/vrf/listener_v1.go [hunk 1]
```diff
@@ -375,7 +382,15 @@ func (lsn *listenerV1) ProcessRequest(req *solidity_vrf_coordinator_interface.VR
 		}
 		return nil
 	}); err != nil {
-		lsn.l.Errorw("Failed executing run", "err", err)
+		lsn.l.Errorw("Failed executing run",
+			"err", err,
+			"reqID", hex.EncodeToString(req.RequestID[:]),
+			"reqTxHash", req.Raw.TxHash)
+	} else {
+		lsn.l.Infow("Executed fulfillment run",
```
Comment (samsondav) [inline core/services/vrf/listener_v1.go:390]: This is going to be way too noisy, can we downgrade to Debug please

File: core/services/vrf/listener_v1.go [hunk 2]
```diff
@@ -274,6 +274,13 @@ func (lsn *listenerV1) handleLog(lb log.Broadcast, minConfs uint32) {
 	})
 	lsn.reqAdded()
 	lsn.reqsMu.Unlock()
+	lsn.l.Infof("Enqueued randomness request",
```
Comment (samsondav) [inline core/services/vrf/listener_v1.go:277]: Downgrade to Debug here please

**Review by samsondav — CHANGES_REQUESTED** [review]

File: core/services/vrf/listener_v1.go (hunk 1 above)
Comment (makramkd) [reply core/services/vrf/listener_v1.go:390]: I think will have to disagree, we have this for V2 and it's definitely useful.

Comment (Nic0S) [reply core/services/vrf/listener_v1.go]: Done

File: core/services/vrf/listener_v1.go (hunk 2 above)
Comment (Nic0S) [reply core/services/vrf/listener_v1.go]: Done

**Review by makramkd — APPROVED** [review]

**Pull request merged**